### PR TITLE
chore: esbuild

### PR DESCRIPTION
**Pull request by mleralec**
## Local
**chore-esbuild:** 
`time yarn build:packages  20,37s user 6,49s system 233% cpu 11,489 total`

**master: (with rollup)**
`time yarn build:packages  116,30s user 17,81s system 257% cpu 52,006 total`

Almost 6 times faster with esbuild :rocket:

## CI
**master:**
`lerna success run Ran npm script 'build' in 68 packages in 27.2s:`

**chore-esbuild:**
`lerna success run Ran npm script 'build' in 68 packages in 4.1s:`

Almost 7 times faster with esbuild 💥

### CODE REVIEW

**Review by cnairiwttj — APPROVED** [review]
Wow congrats champion 🐰

File: rollup.config.js [hunk 1]
```diff
@@ -1,59 +0,0 @@
...
-  plugins: ['babel-plugin-annotate-pure-calls', 'babel-plugin-transform-async-to-promises'],
-  babelHelpers: 'inline',
-})
-
-const external = id => !id.startsWith('.') && !id.startsWith('/')
-
-const PLUGINS = [
-  replace({
-    __BRANCH__: process.env.BRANCH,
-    __ICON_FONT_HASH__: process.env.ICON_FONT_HASH,
```
Comment (theo-mesnil) [inline rollup.config.js:24]: Tu as réussi à gérer différemment la partie font ?

Comment (mleralec) [reply rollup.config.js:24]: https://github.com/WTTJ/welcome-ui/pull/1897/commits/69374aea3a207595f23a384e1d33ef7d6fee8ed6 😅

**Review by theo-mesnil — APPROVED** [review]

File: rollup.config.js (hunk 1 above)
Comment (theo-mesnil) [reply rollup.config.js:24]: 🙈

**Pull request merged**